aesara/tensor/basic.py

```diff
@@ -341,8 +341,13 @@ def get_scalar_constant_value(
 
             if isinstance(data, np.ndarray):
                 return numpy_scalar(data).copy()
-            else:
-                return data
+
+            from aesara.sparse.type import SparseType
```

Comment:
Could a whitelist of supported types be an alternative here?
After all that import is sort of cyclic...

Reply:
Given the nature of this function (i.e. `get_scalar_constant_value`), it should also be converted to a dispatch function.  That way, we can keep irrelevant types out of `aesara.tensor.basic` entirely (e.g. `Assert`).

Reply:
Actually, we have an issue for this: #96.